prog/vm/nexus.rb

```diff
@@ -532,7 +532,7 @@ def write_params_json
 
     unless host.nil?
       begin
-        host.sshable.cmd("sudo timeout 10s systemctl stop #{q_vm}")
+        host.sshable.cmd("sudo systemctl stop #{q_vm}")
```

Comment:
This drops the 10 second limit and uses the default limit. Is that what we want? If not:

```suggestion
        host.sshable.cmd("sudo systemctl stop #{q_vm}", timeout: 10)
```